### PR TITLE
added nextra blog

### PR DESCRIPTION
**Pull request by prathamdupare**

### CODE REVIEW

**Comment by prathamdupare** [comment]
- Blog pages are contained in ```/pages``` directory
- Visit the blog on ```/blog/june```

File: frontend/next.config.mjs [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+  images: {
+    domains: ['ap-south-1.graphassets.com'],
+  },
+  typescript: {
+    // !! WARN !!
+    // Dangerously allow production builds to successfully complete even if
+    // your project has type errors.
+    // !! WARN !!
+    ignoreBuildErrors: true,
+  },
```
Comment (krishnaacharyaa) [inline frontend/next.config.mjs:15]: We can't have this, we need to remove this

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
If you could nicely tell us how to use this that would be great

How to view and how to insert

File: frontend/next.config.mjs (hunk 1 above)
Comment (prathamdupare) [reply frontend/next.config.mjs:15]: Oh I will remove it, it's because I replaced this config with one of my project's.

**Comment by prathamdupare** [comment]
image


This pull request integrates Nextra to support writing and managing blogs within our Next.js application. The changes include:

- Added a `blog` directory under the `pages` directory to store blog posts.
- Blog posts can be written in `.mdx` format. An example post `june.mdx` has been added.
- Included `_meta.json` file to specify the order and titles of the blog posts for Nextra.

Directory structure:
frontend/pages
├── blog 
 │   └── june.mdx
└── _meta.json

For additional info, refer to Nextra's official [documentation](https://nextra.site/docs).

**Pull request merged**